packages/core/src/components/forms/_common.scss

```diff
@@ -56,27 +56,33 @@ $control-group-stack: (
 );
 
 // animating shadows requires the same number of shadows in different states
-@function input-transition-shadow($color: $input-shadow-color-focus, $focus: false) {
+@function input-transition-shadow(
+  $color: $input-shadow-color-focus,
+  $focus: false,
+  $outer-shadow-focus-width: 1px,
+) {
   @if $focus {
     @return
-      inset border-shadow(1, $color, 1px),
-      border-shadow(0.3, $color, 2px);
+      inset border-shadow(0.752, $color, 1px),
+      border-shadow(0.752, $color, $outer-shadow-focus-width);
```

Comment:
border-shadow width stayed the same in the case of intent shadows (since they already start with 1px in un-focused state, the focus state is a 3px shadow outline)

for non-intent inputs, the shadow width reduced by 1px. I expect this to mean that shadows that worked in previous situations will continue to work (ex there won't be clipping introduced from something like a new outline offset or wider shadow)